config/application.rb

```diff
@@ -41,6 +41,11 @@ class Application < Rails::Application
     # the I18n.default_locale when a translation can not be found)
     config.i18n.fallbacks = true
 
+    # For deployment through an iframe as part of the europeana portal, the application needs to load a JS file
+    # The value provided here is not a stable version, you should specify a fixed version in:
+    # config/initializers/e7a_1418_iframe.rb
+    config.e7a_1418_iframe_source = 'http://styleguide.europeana.eu/js/modules/eu/e7a_1418_iframe.js'
```

Comment:
Should this rather just set the base URL like "http://styleguide.europeana.eu" or "https://europeana-styleguide-production.s3.amazonaws.com/v0.2.88", with the asset sub-path, "/js/modules/eu/e7a_1418_iframe.js" in the template?

Because the sub-path should not change per-deployment, just the base URL.